daemon/cmd/daemon_main.go

```diff
@@ -798,6 +798,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Duration(option.CTMapEntriesTimeoutSYNName, 60*time.Second, "Establishment timeout for entries in TCP CT table")
 	option.BindEnv(vp, option.CTMapEntriesTimeoutSYNName)
 
+	flags.Duration(option.CTMapEntriesTimeoutClosingName, 8000*time.Second, "Closing Timeout for entries in TCP CT table")
```

Comment:
Is there specific guidance or motivation behind the 8000s (~2h 13m 20s) timeout here? Did this come from an RFC somewhere? I see earlier iterations of the PR had much lower timeouts, but I couldn't find any discussion around the selection of the default.